apiserver/apiserver.go

```diff
@@ -1182,6 +1195,7 @@ func (srv *Server) GetAuditConfig() auditlog.Config {
 }
 
 // GetCentralHub returns the central hub for the server.
+// TODO (stickupkid): Remove me. This is only used for testsing.
 func (srv *Server) GetCentralHub() *pubsub.StructuredHub {
 	return srv.shared.centralHub.(*pubsub.StructuredHub)
 }
```

Comment:
This is only used for testing, once we remove the peergrouper, this can die.

Reply:
`testsing`